python-selenium/Dockerfile

```diff
@@ -6,7 +6,7 @@ LABEL maintainer="support@apify.com" Description="Base image for Apify actors wr
 
 # Install firefox and gecko driver
 RUN apt-get update \
- && apt-get install -y --no-install-recommends ca-certificates curl firefox-esr \
+ && apt-get install -y --no-install-recommends ca-certificates curl firefox-esr unzip jq \
  && curl -L https://github.com/mozilla/geckodriver/releases/download/v0.33.0/geckodriver-v0.33.0-linux64.tar.gz | tar xz -C /usr/local/bin
```

Comment:
I wanted to suggest to also update `geckodriver`, but wow, it has no new releases in the past 6 months 😄